typewise-alert.c

```diff
@@ -1,71 +1,61 @@
 #include "typewise-alert.h"
 #include <stdio.h>
 
+BreachType breachType;
+
+int Battery_TempLimit[NUMBER_OF_BATTERY_TYPE][6] = 
```

Comment:
this is a 2d array with 18 entries. Please correct me if i am wrong. The requirement is for only 6 values. Please check

Reply:
yes. I used 2D array because in future when we adding new battery type it will easy to increase the no.of rows

Reply:
Sorry, my question was there are 3 rows with 6 columns each. Whereas the requirement is for 2 columns only. So i asked whether 6 columns is fine or not. I understand the use for 2d array 👍

Reply:
Present array is not for 18 entries. Here  NUMBER_OF_BATTERY_TYPE enum value is 1, so there is 1 row and 6 column total 6 entries. 
In this implementation i need 6 columns to update the breach higher and lower limit.

Reply:
Clear👍